.github/workflows/github-actions-docs-test.yml

```diff
@@ -7,28 +7,34 @@ jobs:
   docs-test-job:
     name: 'Test docs for Tcl syntax and README'
     runs-on: ubuntu-latest
+    container:
+      image: openroad/ubuntu-cpp20
 
     steps:
     - name: Checkout repository
       uses: actions/checkout@v3
       with:
-        submodules: true
+        submodules: 'recursive'
 
     - name: Install required package
       run: |
-        sudo apt-get update && sudo apt-get install -y pandoc
-    - name: Set up Python
-      uses: actions/setup-python@v5
-      with:
-        python-version: '3.10'  
+        apt-get update 
+        apt-get install -y pandoc
+
+    - name: Build OpenROAD
+      run: |
+        ./etc/Build.sh -compiler='clang-16' -cmake='-DCMAKE_BUILD_TYPE=Debug -DCMAKE_CXX_STANDARD=20'
```

Comment:
Do you actually need to compile OR in order to generate the docs?  I'm wondering if there is docs target.

Reply:
We just need the `messages.txt` generated as part of the compilation. I can simulate it with a script i suppose